README.md

```diff
@@ -1,3 +1,5 @@
 # ACHP
-A python-based air conditioning and heat pump system model
+A python-based air conditioning and heat pump system model v1.5
+
+This version uses CoolProps v5.x
```

Comment:
No worries, I see that a lot.

On Wed, Sep 30, 2015 at 6:58 PM, Ammar Bahman <notifications@github.com>
wrote:

> In README.md <https://github.com/TSTK/ACHP/pull/16#discussion_r40870645>:
>
> > @@ -1,3 +1,5 @@
> >  # ACHP
> > -A python-based air conditioning and heat pump system model
> > +A python-based air conditioning and heat pump system model v1.5
> > +
> > +This version uses CoolProps v5.x
>
> My bad! [image: :sweat_smile:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TSTK/ACHP/pull/16/files#r40870645>.
>